cmake/SetupBlaze.cmake

```diff
@@ -57,6 +57,32 @@ if(SLEEF_FOUND)
   set(_BLAZE_USE_SLEEF 1)
 endif()
 
+# If BLAZE_USE_STRONG_INLINE=ON, Blaze will use this keyword to increase the
+# likelihood of inlining. If BLAZE_USE_STRONG_INLINE=OFF, uses inline keyword
+# as a fallback.
+option(BLAZE_USE_STRONG_INLINE "Increase likelihood of Blaze inlining." ON)
+
+set(_BLAZE_USE_STRONG_INLINE 0)
+
+if(BLAZE_USE_STRONG_INLINE)
+  set(_BLAZE_USE_STRONG_INLINE 1)
+endif()
+
+message(STATUS "BLAZE_USE_STRONG_INLINE: ${BLAZE_USE_STRONG_INLINE}")
```

Comment:
I'd say delete this line and the one below, since we also don't print out all other config flags. These options are written in the CMakeCache.txt file if needed for debugging.

Reply:
Done!